5.0/en/0x05-For-Users-Of-4.0.md

```diff
@@ -39,7 +39,7 @@ Efforts have also been made to ensure that chapters and sections are organized a
 
 This restructuring has led to the creation of additional chapters:
 
-* **Self-contained Tokens** – Previously included under session management, the expanded use cases and associated security risks of self-contained tokens warranted a dedicated chapter, with many requirements newly introduced in version 5.x.
+* **Self-contained Tokens** – Formerly grouped under session management, self-contained tokens are now recognized as a distinct mechanism and a foundational element for stateless communication (such as in OAuth and OIDC). Due to their unique security implications, they are addressed in a dedicated chapter, with many new requirements introduced in version 5.x.
```

Comment:
> with many new requirements introduced in version 5.x

"many" and "new" - it is the smallest chapter, with 7 requirements in total, 6 of them are new.

This part can be removed, or just remove the word "many".

Reply:
Let's say some instead